### DISCUSSION TITLE
Taro3.1.4 百度小程序编译后报错 Error: deviceRatio 配置中不存在 750 的设置

### DISCUSSION THREAD

**Discussion by paulloo** [NervJS/taro#8988] [Q & A] [answered]
index.js 配置
```

  deviceRatio: {
    640: 2.34 / 2,
    750: 1,
    828: 1.81 / 2
  },
```
到百度小程序这里 deviceRatio 变成了空对象 ｛｝
![image](https://user-images.githubusercontent.com/32083925/112973077-8307c580-9183-11eb-8c5a-107b8d7a6a27.png)

**Comment by paulloo** [comment, marked as answer]
编译H5正常

**Reply by honlyHuang** [reply]
这个没有遇见过哦，你搞个 demo 提个 issue咯

**Reply by paulloo** [reply]
找到原因了，自己写的组件库 加了行代码 Taro.initPxTransform  deviceRatio 没赋值。
